test/integration/vue.spec.ts

```diff
@@ -21,22 +38,59 @@ describe.each([/*[true], */ [false]])(
         pluginOptions: { ...options.pluginOptions, useTypescriptIncrementalApi }
       });
 
-    it('should create a Vue program config if vue=true', async () => {
-      const { getKnownFileNames, files } = await createCompiler({
-        pluginOptions: { vue: true }
-      });
+    it.each(templateCompilers)(
```

Comment:
Please do not wrap every single test in `it.each` - rather wrap the whole suite in `describe.each`.If a test really should run in only one of those two cases, you can still use something similar to skipIfIncremental from the general test helpers.

Reply:
My point was that we need to reduce the number of unnecessary test runs, but I got your point.
```skipIfIncremental``` is interesting solution, I will use it.

Reply:
Done.
I did not applied prettier for ```.test/integration/vue.spec.ts``` to simplify code review.
I will apply it later.
```// eslint-disable-next-line @typescript-eslint/array-type``` was added, because prettier can't parse ts generics.